tilavarauspalvelu/settings.py

```diff
@@ -376,7 +376,7 @@
 
 LOGIN_REDIRECT_URL = "/admin/"
 LOGOUT_REDIRECT_URL = "/admin/"
-SESSION_SERIALIZER = "tilavarauspalvelu.utils.serializers.PickleSerializer"
+SESSION_SERIALIZER = "helusers.sessions.TunnistamoOIDCSerializer"
```

Comment:
How does this affect current sessions? Do you get logged out or does it just work? Or are there complications?

Reply:
Should not affect current sessions at all (tested only locally)